android/build.gradle

```diff
@@ -16,5 +16,5 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.11.+'
+    compile 'com.facebook.react:react-native:0.33.+'
```

Comment:
Does this work? The latest version should be 0.20.1 according to https://mvnrepository.com/artifact/com.facebook.react/react-native. PR #3 seems to be the correct fix. Note also that these changes have been required since RN 0.29.0.